pkg/cluster/context.go

```diff
@@ -207,3 +211,17 @@ func (c *Context) CollectLogs(dir string) error {
 	}
 	return logs.Collect(nodes, dir)
 }
+
+type contextConfig struct {
+	network string
+}
+
+// Option allows optional configuration of the cluster context.
+type Option func(*contextConfig)
+
+// WithNetwork sets the container network to be used when creating a cluster.
+func WithNetwork(network string) func(*contextConfig) {
```

Comment:
is there any reason this should be an option on context instead of Create()? do you forsee other calls needing this?

:+1: for this pattern in general.

Reply:
I didn't know if anything else was calling Create so I didn't want to change the parameters. Either way works. I haven't dug into the code enough to see how/where it is used.